data-raw/weather.R

```diff
@@ -8,17 +8,23 @@ library(readr)
 # Download ---------------------------------------------------------------------
 
 get_asos <- function(station) {
-  url <- "https://mesonet.agron.iastate.edu/cgi-bin/request/getData.py"
+  url <- "http://mesonet.agron.iastate.edu/cgi-bin/request/asos.py?"
   query <- list(
     station = station, data = "all",
     year1 = "2013", month1 = "1", day1 = "1",
     year2 = "2013", month2 = "12", day2 = "31", tz = "GMT",
     format = "comma", latlon = "no", direct = "yes")
-
+  
   r <- GET(url, query = query)
   stop_for_status(r)
-
-  writeBin(content(r, "raw"), paste0("data-raw/weather/", station, ".csv"))
+  if(!dir.exists("data-raw/weather")){
+    if(!dir.exists("data-raw"))
+      dir.create("data-raw")
+    
+    dir.create("data-raw/weather")
+  }
```

Comment:
I chose this more verbose option because of the caution in the Details section of `dir.create`

>  One of the idiosyncrasies of Windows is that directory creation may report success but create a directory with a different name

But happy to change.